### ISSUE TITLE
Add section of the docs to list projects that are built using Perk

### ISSUE THREAD

**Issue by alarner** [alarner/perk-docs#11]

**Comment by hemangsk** [comment]
Hi @alarner 😄 
I would like to work on this issue 😃  😄  can you guide me...

**Comment by alarner** [comment]
Hey @hemangsk! Thanks for offering to help out! I'm not sure we have enough Projects yet that are using Perk to Justify that page. Is there another issue that perhaps you'd be interested in? Alternatively you could put together the page with some fake projects and once we reach a critical mass of projects we could add them in and link to it on the docs navigation.

**Comment by hemangsk** [comment]
@alarner I am really excited and looking ahead to work on this issue and other issues/projects on the perk as well 😄  😃 
Can I start with this one first.. creating a 'projects using Perk' page that follows the Perk design philosophy, with dummy content and link it to the docs?

**Comment by hemangsk** [comment]
A  timeline kind of I've just come up with, 
- [ ] Mockup of the new page
- [ ] Changes on the mockup
- [ ] Code the final thing

**Comment by alarner** [comment]
@hemangsk sounds great!

**Comment by hemangsk** [comment]
@alarner  I am facing an issue setting the website up. In the builds directory , the links aren't being properly generated. 
They are generated like, 
<main class="row nav-padding">
        <nav class="four columns topics">

```
    <a href="/v1/video-reel/" class="list-item active">Video Reel</a>

    <a href="/v1/video-reel/" class="list-item active">01 - Up &amp; Running</a>

    <a href="/v1/video-reel/" class="list-item active">02 - Modifying the Homepage View</a>

    <a href="/v1/video-reel/" class="list-item active">03 - Database Migrations - Creating Tables</a>

    <a href="/v1/video-reel/" class="list-item active">04 - Setting up an API</a>

    <a href="/v1/video-reel/" class="list-item active">05 - Using Sass, ES6 and Modules</a>

    <a href="/v1/video-reel/" class="list-item active">06 - Setting up a Single Page App</a>`
```

can you please help me figure out the problem ?
Thanks!

**Comment by alarner** [comment]
@hemangsk that's a strange problem. Is this happening to all of the links, or just the ones on the video navigation? Are you running on Windows, Mac or Linux?

**Comment by hemangsk** [comment]
Its happening to links in the  'topics lists' only, I am on Mac.

**Comment by hemangsk** [comment]
I cloned the repository and did step 1 - npm install and then step 2- npm start. Is there any intermediate step which I am missing?

**Comment by alarner** [comment]
No that should be it. Are you seeing any errors in the terminal?

**Comment by alarner** [comment]
My first thought is that it's some sort of bug in this custom plugin that I wrote to generate those link urls. I'm having a hard time debugging though because I can't reproduce it.

https://github.com/alarner/perk-docs/blob/master/plugins/path.js

**Comment by hemangsk** [comment]
Sounds like we have the culprit. 
I tried to use console log statements in topics.html file

```
<nav class="four columns topics">
    <%
        var group = collection[0] === 'group' ? collection[1] : collection[0];
        var topics = collections[group];
    %>
    <% topics.forEach(function(topic) {

        console.log(topic.filename); // Stat1
        console.log(topic.path);  //Stat2
        %>
        <a href="<%= topic.path %>" class="list-item<%= topic.path === path ? ' active' : '' %>"><%= topic.title %></a>
    <% }); %>
</nav>
```

`console.log(topic.filename);` is giving output as v1/video-reel/04-setting-up-an-api.md,  the 'md' extension being not changed to 'html' , and the custom plugin also seems to be doing something related to this extension handling and management. 

`console.log(topic.path)` outputs /v1/video-reel/ for every item in topic list. 

Same thing is happening for guides, common errors and apis, so do we have to customise the plugin ?

**Comment by alarner** [comment]
Can you try putting a console.log inside of the plugin at a couple points and see what type of output you get?

Also another thing occurred to me, which version of node are you using?

``` js
var path = require('path');
module.exports = function(opts){

    // commonly set default options here

    // return the function to be given to the `.use` call.
    return function(files, metalsmith, done){
        // ...do something with `files` here...
        var f = [];
        for(var i in files) {
            var file = files[i];
                        console.log(1, file);
            if(!file.filename) {
                continue;
            }
            var parsed = path.parse(file.filename);
                        console.log(2, parsed);
            if(parsed.ext.toLowerCase() === '.md') {
                parsed.ext = '.html';
                delete parsed.base;
                file.path = path.join('/', path.format(parsed));
                                console.log(4, file.path);
            }
            else {
                file.path = file.filename;
                                console.log(5, file.path);
            }
        }
        done();
    };
};
```

**Comment by hemangsk** [comment]
Its node v5.50,  trying out the console log statements in plugin

**Comment by hemangsk** [comment]
Here it is,

```
1 { title: 'Video Reel',
  description: 'Tutorial videos on how to use Perk - Passport, Express, Redis and Knex.',
  date: Sun Jul 17 2016 05:30:00 GMT+0530 (IST),
  layout: 'topic-list.html',
  group: 'video-reel',
  order: 4,
  contents: <Buffer 3c 70 3e 54 75 74 6f 72 69 61 6c 20 76 69 64 65 6f 73 20 6f 6e 20 68 6f 77 20 74 6f 20 75 73 65
 20 50 65 72 6b 20 2d 20 50 61 73 73 70 6f 72 74 2c 20 ... >,
  mode: '0644',

  stats:
   { dev: 16777220,
     mode: 33188,
     nlink: 1,
     uid: 501,
     gid: 20,
     rdev: 0,
     blksize: 4096,
     ino: 15966658,
     size: 253,
     blocks: 8,
     atime: Sat Oct 01 2016 19:44:44 GMT+0530 (IST),
     mtime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST),
     ctime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST),
     birthtime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST) },
  filename: 'v1/video-reel/index.md',
  collection: [ 'group', 'video-reel' ],
  next:
   { title: '01 - Up & Running',
     description: 'Build a LinkedIn clone using node and express. In this video I give you and overview of the proj
ect that we\'re about to create as well as help you quickly get through that setup work so that you can get to the
fun stuff faster.',
     date: Sun Jul 17 2016 05:30:00 GMT+0530 (IST),
     id: 'au3Ho_E5pak',
     layout: 'reel.html',
     order: 100,

     contents: <Buffer 3c 68 32 20 69 64 3d 22 72 65 73 6f 75 72 63 65 73 22 3e 52 65 73 6f 75 72 63 65 73 3c 2f 68
 32 3e 0a 3c 75 6c 3e 0a 3c 6c 69 3e 3c 61 20 68 72 65 66 ... >,
     mode: '0644',
     stats:
      { dev: 16777220,
        mode: 33188,
        nlink: 1,
        uid: 501,
        gid: 20,
        rdev: 0,    
        blksize: 4096,
        ino: 15966652,
        size: 772,
        blocks: 8,
        atime: Sat Oct 01 2016 19:44:44 GMT+0530 (IST),
        mtime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST),
        ctime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST),
        birthtime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST) },
     filename: 'v1/video-reel/01-up-and-running.md',
     collection: [ 'video-reel' ],
     previous: [Circular],
     next:
      { title: '02 - Modifying the Homepage View',
        description: 'In this video we\'re going to learn how to modify the home page and also explore the perk con
figuration system.',

        date: Sun Jul 17 2016 05:30:00 GMT+0530 (IST),
        id: 'R541RpDR4gs',
        layout: 'reel.html',
        order: 200,
        contents: <Buffer 3c 68 32 20 69 64 3d 22 72 65 73 6f 75 72 63 65 73 22 3e 52 65 73 6f 75 72 63 65 73 3c 2f
 68 32 3e 0a 3c 75 6c 3e 0a 3c 6c 69 3e 3c 61 20 68 72 65 66 ... >,
        mode: '0644',
        stats: [Object],
        filename: 'v1/video-reel/02-modifying-the-homepage-view.md',
        collection: [Object],
        previous: [Circular],
           next: [Object],
        path: '/v1/video-reel/' },
     path: '/v1/video-reel/' },
  previous:
   { title: 'Common Errors',
     description: 'A list of common errors and how to solve them.',
     layout: 'topic-list.html',
     group: 'common-errors',
     order: 3,
     contents: <Buffer 3c 70 3e 43 6f 6d 6d 6f 6e 20 65 72 72 6f 72 73 3c 2f 70 3e 0a>,
     mode: '0644',
     stats:
      { dev: 16777220,
        mode: 33188,
        nlink: 1,

        uid: 501,
        gid: 20,
        rdev: 0,
        blksize: 4096,
        ino: 15966634,
        size: 159,
        blocks: 8,
        atime: Sat Oct 01 2016 19:44:44 GMT+0530 (IST),
        mtime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST),
        ctime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST),
        birthtime: Sat Oct 01 2016 19:27:31 GMT+0530 (IST) }, 
      filename: 'v1/common-errors/index.md',
     collection: [ 'common-errors', 'group' ],
     next: [Circular],
     previous:
      { title: 'API Documentation',
        description: 'API Documentation for Perk.',
        layout: 'topic-list.html',
        group: 'api',
        order: 2,
        contents: <Buffer 3c 70 3e 50 65 72 6b 20 41 50 49 20 64 6f 63 75 6d 65 6e 74 61 74 69 6f 6e 20 69 73 20 64
 65 73 69 67 6e 65 64 20 74 6f 20 62 65 20 61 20 72 65 66 65 ... >,
        mode: '0644',
        stats: [Object],
        filename: 'v1/api/index.md',
        collection: [Object],

        next: [Circular],
        previous: [Object],
        path: '/v1/api/' },
     path: '/v1/common-errors/' } }
2 { root: '',
  dir: 'v1/video-reel',
  base: 'index.md',
  ext: '.md',
  name: 'index' }
4 '/v1/video-reel/'
```

**Comment by hemangsk** [comment]
@alarner I just upgraded node to v6.6.0 and now the website is working as expected! 🎉 😄 😄 
My outdated node.js was the real issue 😢
I am really thankful for your time and patience.
Jumping to the show case page mockup now! 😄

**Comment by alarner** [comment]
👍 glad you could figure it out 🎉

**Comment by hemangsk** [comment]
Hey @alarner ! 😄  Mockup 1 is ready 🎉  🎉
So I kind of made a bunch of assumptions, like we can ask other people's Perk project experience and call it call it as "Perk Story". And Perk users can send in their stories through email (or Pull requests or creating new issues) for you to review and add their Perk projects to showcase.  So do these sound any good and can it be developed upon ? 

I've attached the image below...but taking few seconds to load 😅 

![show case](https://cloud.githubusercontent.com/assets/13018570/19044870/b9677d1e-89b3-11e6-8564-573431b98d53.png)

**Comment by alarner** [comment]
@hemangsk Looks great! Let's move forward!
